contracts/flashloan/Instapool/main.sol

```diff
@@ -138,7 +143,7 @@ contract DydxFlashloaner is Helper, ICallee, DydxFlashloanBase {
             require(sub(initialBal, finalBal) <= _dif, "eth-amount-paid-less");
         } else {
             uint256 _decimals = TokenInterface(token).decimals();
-            uint _dif = wmul(convertTo18(_amount, _decimals), 10000000000); // Taking margin of 0.00000001%
+            uint _dif = wmul(convertTo18(_amount, _decimals), 200000000000); // Taking margin of 0.0000002%
```

Comment:
Do you think Aave has the decimals problem as the 1 atoken is always equals to 1 token.

Reply:
We shouldn't have any problem due to that. I copied this logic from the old contract. I think it was for the debt fee on Aave.

Reply:
So no need to change anything on this? Should we proceed with the merge then? Also, I think there's is no issue on the borrow fee on Aave v2. Right?

Reply:
Yeah. we can merge this